environment.yml

```
@@ -0,0 +1,13 @@
+name: mkdocs-asf-theme
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - python=3.9
+  - pip
+  # For packaging and distribution
+  - build
+  - setuptools>=64
+  - setuptools_scm>=8
+  - mkdocs
+  - mkdocs-material>=9,<10
```

We based all the template files on `9.4.2`, so I think we should require at least that version.
```suggestion
  - mkdocs-material>=9.4.2,<10
```

---

Oh yeah, good call.